test/projects/HarnessTemplate/tiapp.xml

```diff
@@ -69,4 +69,5 @@
         <module platform="android">ti.map</module>
         <module platform="iphone">ti.map</module>
     </modules>
+    <sdk-version>12.5.0.GA</sdk-version>
```

Comment:
If you omit `<sdk-version>`, it'll use the latest installed SDK. This is handy so we don't have to remember to bump this.

Reply:
The test failed without a version and that's why I've put 12.5.0 in there and made sure 12.5.0 is installed in the build workflow

Reply:
The Titanium CLI doesn't need a `<sdk-version>`. I built an alloy app without a `<sdk-version>` and it also worked.

I tried running the tests on Windows and they blow up saying `'alloy' is not recognized as an internal or external command`. `alloy` is in the PATH. I haven't dug further into it.

Maybe leave this as `latest`?

Reply:
This is the error when I use latest and remove the `<sdk-version>` again:

![Bildschirmfoto_20241007_104735](https://github.com/user-attachments/assets/a2fb48db-a5c9-42ab-9eba-f21aac6793d6)

```
Error:  "undefined" was found in the "sdk-version" field of your tiapp.xml.
Error:  If you are building with the legacy titanium.py script and are specifying 
Error:  an SDK version as a CLI argument that is different than the one in your 
Error:  tiapp.xml, please change the version in your tiapp.xml file.
```

Reply:
I'll test on Linux when I get a sec.

Reply:
Ok, looks like you backed out the changes, so we're good.